### ISSUE TITLE
wsl triggeres on Examples

### ISSUE THREAD

**Issue by freman** [golangci/golangci-lint#881]
<details><summary>Version of golangci-lint</summary>

```bash
$ golangci-lint --version
golangci-lint has version 1.20.1 built from 849044b on 2019-10-15T19:11:27Z
```

</details>

<details><summary>Config file</summary>

```bash
$ cat .golangci.yml
```

</details>

<details><summary>Go environment</summary>

```bash
$ go version && go env
go version go1.13.3 darwin/amd64
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/shannon/Library/Caches/go-build"
GOENV="/Users/shannon/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GONOPROXY=""
GONOSUMDB=""
GOOS="darwin"
GOPATH="/Users/shannon/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/darwin_amd64"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/08/9qf64r3d209cvmgznzj49zg40000gn/T/go-build732873448=/tmp/go-build -gno-record-gcc-switches -fno-common"
```

</details>

<details><summary>Verbose output of running</summary>

```bash
$ golangci-lint run -v
INFO [config_reader] Config search paths: [./ /Users/shannon/wsl /Users/shannon /Users /]
INFO [lintersdb] Active 10 linters: [deadcode errcheck gosimple govet ineffassign staticcheck structcheck typecheck unused varcheck]
INFO [loader] Go packages loading at mode 575 (name|types_sizes|files|imports|compiled_files|deps|exports_file) took 122.126171ms
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 182.891µs
WARN [runner] Can't run linter goanalysis_metalinter: SA6001: failed prerequisites: buildssa@_/Users/shannon/wsl_test [_/Users/shannon/wsl.test]
WARN [runner] Can't run linter unused: buildssa: analysis skipped: errors in package: [/Users/shannon/wsl/a_test.go:6:14: undeclared name: stringutil]
INFO [runner] processing took 4.063µs with stages: max_same_issues: 1.13µs, skip_dirs: 499ns, cgo: 334ns, nolint: 288ns, max_from_linter: 261ns, filename_unadjuster: 233ns, path_prettifier: 193ns, diff: 184ns, identifier_marker: 161ns, autogenerated_exclude: 159ns, skip_files: 149ns, source_code: 87ns, max_per_file_from_linter: 80ns, exclude: 79ns, path_shortener: 78ns, uniq_by_line: 74ns, exclude-rules: 74ns
INFO [runner] linters took 94.663383ms with stages: goanalysis_metalinter: 58.446952ms, unused: 36.160888ms
INFO File cache stats: 0 entries of total size 0B
INFO Memory: 4 samples, avg is 69.6MB, max is 69.9MB
INFO Execution took 233.408883ms
```

</details>

Getting `block should not end with a whitespace (or comment) (wsl)` for Example functions but ending the function with a comment is basically how it's meant to be see: https://blog.golang.org/examples#TOC_3.

wsl.go
```go
package wsl

import "fmt"

func WSL() {
	fmt.Println("hello")
}
```

wsl_test.go
```go
package wsl

func ExampleWSL() {
	WSL()

	// Output: hello
}
```
`go test -v`
```
=== RUN   ExampleWSL
--- PASS: ExampleWSL (0.00s)
PASS
ok  	_/Users/shannon/wsl	0.006s
```

`golangci-lint run --enable wsl`
```
wsl_test.go:7:1: block should not end with a whitespace (or comment) (wsl)
}
^
```

Perhaps before complaining about wsl a check could be made to see if it's a `_test` file and an `Example` function?

**Comment by tpounds** [comment]
@freman I'm not sure this is an actual issue as this is a check implemented by wsl. I'm inclined to mark this as "won't fix" since it's a specific check implemented by the linter and is something you can ignore/suppress. @bombsimon Any thoughts here since you are the author of wsl?

**Comment by bombsimon** [comment]
This was a bug and fixed in [1.2.5](https://github.com/bombsimon/wsl/releases/tag/v1.2.5), see [this issue](https://github.com/bombsimon/wsl/issues/38). This should be included in the latest version of golangci-lint but I’ll have to double check in a few hours when I’m by my computer. 

Either way this should absolutely be allowed and not give a warning!

**Comment by bombsimon** [comment]
The ticket states that this was with golangci-lint 1.20.1 but wsl 1.2.5 was merged in 1.20.0. 🤔 

However, the somewhat good news is that I cannot reproduce this so I guess it's not a problem.

```sh
$ golangci-lint --version
golangci-lint has version 1.22.1 built from da91430 on 2019-12-30T15:46:49Z
```

```sh
$ tail wsl{,_test}.go
==> wsl.go <==
package wsl

import "fmt"

func WSL() {
        fmt.Println("hello")
}

==> wsl_test.go <==
package wsl

func ExampleWSL() {
        WSL()

        // Output: hello
}
```

```sh
go test ./... -v
=== RUN   ExampleWSL
--- PASS: ExampleWSL (0.00s)
PASS
ok      _/home/simon/x   0.009s

```

```sh
$ wsl -w ./... && echo $?
0

$ golangci-lint run --no-config --enable wsl ./... && echo $?
0
```

**Comment by tpounds** [comment]
@bombsimon Thanks for confirming!

@freman I'm going to resolve but please report back if you continue to experience issues with the latest release.
